README.rst

```diff
@@ -117,6 +117,18 @@ For instance, ``has_attrs(name="bob")`` is equivalent to ``has_attrs(name=equal_
           "b": equal_to(4),
       }))
 
+* ``is_mapping_with(**matchers)``: matches a mapping, such as a ``dict``, if it has the same keys with matching values.
```

Comment:
I think the arguments here are `matchers` rather than `**matchers` (I made a mistake, now fixed, when documenting `is_mapping()`)

Reply:
Good point, modified.